ui/src/pages/Play.tsx

```diff
@@ -1,42 +1,35 @@
-import React, {useState} from 'react';
-import DrawingCanvas from '../components/DrawingCanvas';
-import navBar from "../components/navbar"
 import './Play.css'
+import React, {useState} from 'react';
+import NumberCard from '../components/NumberCard';
 
 const PlayPage = () => {   
-    const [canvasURI, setCanvasURI] = useState("")   
-    const [number, SetNumber] = useState(0);
-    const [score, SetScore] = useState(0.99);
-        
-    const test = () => {
-        let data = {"image":"23"}
-        fetch("http://127.0.0.1:8000/predict/?" + new URLSearchParams(data)).then(Response => {
-            Response.json().then(Json => {
-                console.log(Json.data);
-                SetNumber(Json.data.classification)
-                SetScore(Json.data.score)
-            });
-        })  
-    }
-    
-    const undo = () => {
-        console.log("clear the screen")
-        SetNumber(0)
-        SetScore(0.99)
-    }
+
 
     return (
         <div className="Play">
-            <div className='container'>
-            <DrawingCanvas setImageURI={setCanvasURI}/>
-                <img className="signature-thumbnail" width="250px" 
-                src ={canvasURI} 
-                alt="Signature thumbnail"></img>
-                <button className="submitButton" onClick={test}>Submit</button>
```

Comment:
className="submit-button"

Reply:
the code is no longer in use, I will update it appropriately.